<ui version="4.0" >
 <class>ExpressionDialog</class>
 <widget class="QDialog" name="ExpressionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>検索条件の編集</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="patternGroupBox" >
     <property name="title" >
      <string>検索文字/正規表現</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="4" >
       <widget class="QLineEdit" name="patternEdit" />
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="matchTypeComboBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>正規表現</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>文字列を含む</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>前方一致</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>完全一致</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QCheckBox" name="matchCaseSensitiveBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>大文字/小文字を区別</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>種類</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>検索対象</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="targetNameBox" >
          <property name="text" >
           <string>配信者</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="targetLongDescriptionBox" >
          <property name="text" >
           <string>詳細</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="targetGenreBox" >
          <property name="text" >
           <string>ジャンル</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="targetPlayingBox" >
          <property name="text" >
           <string>現在の曲</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="targetContactUrlBox" >
          <property name="text" >
           <string>コンタクトURL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="targetTypeBox" >
          <property name="text" >
           <string>種類</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>評価点数</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="pointSpinBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum" >
        <number>-99</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>patternGroupBox</tabstop>
  <tabstop>patternEdit</tabstop>
  <tabstop>matchTypeComboBox</tabstop>
  <tabstop>matchCaseSensitiveBox</tabstop>
  <tabstop>targetNameBox</tabstop>
  <tabstop>targetLongDescriptionBox</tabstop>
  <tabstop>targetGenreBox</tabstop>
  <tabstop>targetPlayingBox</tabstop>
  <tabstop>targetContactUrlBox</tabstop>
  <tabstop>targetTypeBox</tabstop>
  <tabstop>pointSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExpressionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExpressionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patternGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>44</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel" >
     <x>39</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patternGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pointSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel" >
     <x>118</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patternGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>18</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel" >
     <x>23</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
